config/default.yml

```diff
@@ -202,6 +202,11 @@ Rails/EnvironmentComparison:
   Enabled: true
   VersionAdded: '0.52'
 
+Rails/Exclude:
+  Description: 'Favor `collection.exclude?(obj)` over `!collection.include?(obj)`.'
```

Comment:
Although `Favor` is used in a few places, `Prefer` is much more common, so I'd suggest we use that.

Reply:
No problems, changed.